gax/src/main/java/com/google/api/gax/batching/v2/BatcherImpl.java

```diff
@@ -170,8 +195,10 @@ private void awaitAllOutstandingBatches() throws InterruptedException {
   /** {@inheritDoc} */
   @Override
   public void close() throws InterruptedException {
-    isClosed = true;
+    if (isClosed) return;
```

Comment:
Please use braces for the body:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used